sections/algorithmic_components.tex

```diff
@@ -544,36 +544,43 @@ \subsubsection{Single Visit}
-\item Fit zeropoint and possibly perturbations to all CCDs on multiple visits simultaneously after matching to reference catalog.
+\item Deriving SEDs for calibration stars from colors and reference catalog classifications.
+\item Utilizing additional information from wavelength-dependent photometric calibration built by calibration products production to convert observed flux to a flux through a standard atmosphere.
+\item Fitting zeropoint and possibly perturbations to all CCDs on multiple visits simultaneously after matching to reference catalog.
 \end{itemize}
 
-Because the number of stars gets ridiculously large, it can be useful to solve different overlapping regions of the sky in parallel.  Once all the regions converge, you can run the same ubercal matrix solution to tie the patches together.  The fist step is to solve:
+The first step is to solve:
 \begin{equation} \label{eq:uber}
-m_{ij} = m_i + z_j
+m_{i,j} = m_i + z(x, y)_j
 \end{equation}
-where $m_{ij}$ is an observed magnitude of star with true magnitude $m_i$ on observation $j$.  While it is possible to include more terms (say, fit out flat-fielding errors at the same time), more terms makes it much harder to solve in parallel.  Additional terms also make it easy to make the problem degenerate and it can slow the ubercal algorithm down rapidly. Naively, solving Equation~\ref{eq:uber} would involve simply computing the psudo-inverse of the sparse $m_{ij}$ matrix. Unfortunately, the inverse of a large sparse matrix is a large dense matrix.  Thus one must use iterative solvers such as the LSQR algorithm (a conjugate gradient-type solver) to find the best-fitting values of $m_i$ and $z_j$.  
+where $m_{i,j}$ is an observed magnitude of star with true magnitude $m_i$ on observation $j$, with position-dependent photometric scaling $z$.   Naively, solving Equation~\ref{eq:uber} would involve simply computing the psudo-inverse of the sparse $m_{ij}$ matrix. Unfortunately, the inverse of a large sparse matrix is a large dense matrix.  Thus one must use iterative solvers such as the LSQR algorithm (a conjugate gradient-type solver) to find the best-fitting values of $m_i$ and the (model-dependent) parameters of $z_j$.
+
+\subsubsection{Global Fitting}
+\label{sec:acGlobalPhotometricFit}
+
+Global photometric fitting (as opposed to tract-level \hyperref[sec:acJointPhotometricFit]{Joint Multi-Visit Photometric Fitting}) will most likely not be run in Data Release Production, as we expect to be able to use the Gaia catalogs to calibrate between tracts.  The suitability of the Gaia catalogs for this purpose still needs to be confirmed in detail (it depends on our ability to type stars using the a combination of Gaia BP/RP spectra and LSST colors).  Even if Gais is used, a global photometric fit may still be useful as a QA tool (run after DRP) to verify the quality of the global photometric.
```

Comment:
Not to much type them as predict all the LSST colours -- which may require restricting ourselves to a range of colours which is (I agree) tantamount to typing.  I'm hoping to be able to do this from only the RP/BP spectra, but don't yet know if this is possible.

Also typo "Gais" for Gaia.  "Psudo" for Pseudo